install/installer/traf_config_setup

```diff
@@ -226,6 +230,33 @@ echo "export NODE_LIST=\"$NODE_LIST\"" >> $LOCAL_TRAF_CONFIG
 node_count=$(echo $NODE_LIST | wc -w)
 
 #==============================================
+echo -n "Are management nodes enabled (Y/N), default is N: "
+read answer
+ 
+if [ -z $answer ]; then
+      echo "export MANAGE_ENABLED=\"N\"" >> $LOCAL_TRAF_CONFIG
+else
+      if [[ "${answer}" =~ ^[Yy]$ ]]; then
```

Comment:
The above 3 lines can be eliminated if the default  variable is initialized.. Also instead of MANAGE_ENABLED shall we change it to MANAGEMENT_NODES_ENABLED